astropy/config/tests/test_configs.py

```diff
@@ -129,6 +130,25 @@ def test_config_file():
     reload_config('astropy')
 
 
+def test_generate_config(tmp_path):
+    from astropy.config.configuration import generate_config
+    out = io.StringIO()
+    generate_config('astropy', out)
+    conf = out.getvalue()
+
+    outfile = tmp_path / 'astropy.cfg'
+    generate_config('astropy', outfile)
+    with open(outfile) as fp:
+        conf2 = fp.read()
+
+    for c in (conf, conf2):
```

Comment:
Maybe a comment to explain why these specific lines are selected for testing? Were they arbitrarily selected or were there reasons?

Reply:
Just a random selection :smile:

Reply:
Added a comment.